qlib/kernel/quring/uring_mgr.rs

```diff
@@ -365,15 +387,22 @@ impl QUring {
         srcs: &[IoVec],
         ops: &TsotSocketOperations,
     ) -> Result<i64> {
-        let (count, writeBuf) = buf.Writev(task, srcs)?;
+        #[cfg(feature = "cc")]
```

Comment:
For a placeholder, I suggest leave this function as is (with `#[cfg(not(feature = "cc"))]`), make a cc version of it and make it `todo!()`

The same goes for other places where you have e.g. `panic!("not supported by cc");`. The idea is to make changes minimal, if the cc version is not ready.

Reply:
I see.